task-json/q2.js

```diff
@@ -1,27 +1,22 @@
 

```

Comment:
Here you did the same mistake, this file should be in different branch, not here